util/chplenv/chpl_home_utils.py

```diff
@@ -18,15 +18,20 @@ def get_chpl_home():
     return chpl_home
 
 @memoize
+def get_chpl_module_home():
+    return os.environ.get('CHPL_MODULE_HOME', '')
+
+@memoize
 def get_chpl_third_party():
     default = os.path.join(get_chpl_home(), 'third-party')
     chpl_third_party = overrides.get('CHPL_THIRD_PARTY', default)
     return chpl_third_party
 
 @memoize
 def using_chapel_module():
-    chpl_home = overrides.get('CHPL_HOME', '')
+    chpl_home = os.path.normpath(overrides.get('CHPL_HOME', ''))
```

Comment:
Nah, CHPL_HOME is the only path that overrides.get() supports

We could do the normpath in get_chpl_home() and get_chpl_module_home(), but I don't really think that's needed. It's not really needed here either since the module is what sets CHPL_HOME and CHPL_MODULE_HOME, but I happened to notice this while testing the change